wiki/en/Running-a-Server.md

```diff
@@ -36,86 +37,108 @@ However, various problems can also arise when setting up Servers - especially wh
+1. Make sure you have a current list of standard packages: `sudo apt update`
+1. Install the Jamulus package for your architecture, for example: 
+
+	`sudo apt install ./{{ site.download_file_names.deb-headless }}`
+
+1. Enable the headless Server process: 
+
+	`sudo systemctl enable jamulus-headless`
+
+1. Add your desired [command line options](Running-a-Server#configuration) to the `ExecStart` line in the systemd service file (by default you will be running an Unregistered Server):
 
-### 2. Unregistered
+	`sudo systemctl edit --full jamulus-headless`
 
-This is the default when starting a Server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your Server's address to will be able to connect to it. This is useful because Jamulus does not let you control who can connect to a Server.
+1. Reload the systemd files and restart the headless Server:
+
+	`sudo systemctl daemon-reload && systemctl restart jamulus-headless` 
+
+_To upgrade your Server, just repeat the steps above._
+
+_To amend your Server configuration, just repeat the last two steps above._
+
+---
+
+## Server Modes
+
+Servers can be run in one of three modes (either at home or on a 3rd party host), depending on your needs.
+
+### Unregistered mode
+
+This is the default when starting a Server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your Server's address can connect to it. 
+
+**For information about running an Unregistered Server [see this guide](Unregistered-Servers).**
 
 <figure>
 	<img src="{% include img/en-screenshots/diagram-unreg-server.inc %}" loading="lazy" alt="Diagram of connections between Clients within a Jamulus Unregistered Server">
-	<figcaption>How Unregistered Servers work</figcaption>
 </figure>
 
-If you are running an unregistered server behind a home internet connection, you might need to enable [port forwarding](#port-forwarding) as described below.
+### Registered mode
 
-### 3. Directory
+In this mode your Server will be included in the server list supplied by a Directory. Jamulus has a list of built-in Directories that Clients can know, and if you register your Server with one of these, anyone can then discover and connect to it.
 
-If you want to run a number of Servers, possibly also behind a firewall or on a LAN, you may want to run your Server as a Directory. Examples include online events, music associations, sectional rehearsals or music lessons for schools.
+You can also have your Server listed by a Custom Directory (see below), if that better meets your requirements.  Clients can then only see your Server if they also know the Custom Directory.
+
+Note that Directories can only register up to 150 Servers. If you see a message that says you cannot register your Server becuase the Directory is full, you can try registering on another. 
```

Comment:
```suggestion
Note that Directories can only register up to 150 Servers. If you see a message that says you cannot register your Server because the Directory is full, you can try registering on a different Directory. 
```